xcode/Subconscious/Shared/Components/Common/Forms/ValidatedTextField.swift

```diff
@@ -22,6 +22,10 @@ struct ValidatedFormField<T: Equatable>: View {
     var autoFocus: Bool = false
     var submitLabel: SubmitLabel = .done
     var onSubmit: () -> Void = {}
+    var onFocusChanged: (_ focused: Bool) -> Void = { _ in }
+    
+    /// Indicate if this field should appear invalid due to a side effect failing etc.
+    var isValid: Bool = true
```

Comment:
Does this toggle on/off the validation feature?

Reply:
No, the name might be misleading... but I might just be approaching this the wrong way.

I wanted to mark a field as invalid due to an an error in a side effect... but as I write this I think I could just inline my logic into the regular validation function. Or display a seperate error panel.